tests/integration_tests/ui_tests/ArelleGUITest/ArelleGUITest/Tests.cs

```diff
@@ -0,0 +1,373 @@
+            using (var archive = ZipFile.OpenRead(exampleDocumentPath))
+            {
+                nFilesInDocumentZip = archive.Entries.Count;
+                indexOfEntryPoint = archive.Entries.Select((e, i) => new { e, i }).Where(x => x.e.Name == exampleDocumentEntryPoint).Single().i;
+            }
+            var fileNameComboBox = Retry.WhileNull(
+                () => openFileWindow.FindFirstDescendant(cf => cf.ByControlType(ControlType.ComboBox).And(cf.ByName("File name:")))).Result.AsComboBox();
+            Logger.Default.Info("enter path to document");
+            fileNameComboBox.EditableText = exampleDocumentPath;
+            Wait.UntilInputIsProcessed();
+            Logger.Default.Info("open document");
+            Keyboard.Type(VirtualKeyShort.ENTER);
+            Wait.UntilInputIsProcessed();
+
+            Logger.Default.Info("select file inside document archive");
+            var selectFileFromArchiveWindow = Retry.WhileNull(
+                () => Window.FindFirstChild(Automation.ConditionFactory.ByControlType(ControlType.Window)), timeout: TimeSpan.FromSeconds(3)).Result;
+            var selectFileFromArchivePane = selectFileFromArchiveWindow.FindFirstByXPath("/Pane/Pane/Pane/Pane[1]");
+            Mouse.LeftClick(new Point(
+                (int)(selectFileFromArchivePane.BoundingRectangle.X + selectFileFromArchivePane.BoundingRectangle.Width / 2.0f),
+                (int)(selectFileFromArchivePane.BoundingRectangle.Y
+                // +1 for the header, +0.5 to end up in the middle
+                + selectFileFromArchivePane.BoundingRectangle.Height * (indexOfEntryPoint + 1.5) / (nFilesInDocumentZip + 1))
+            ));
+            Keyboard.Type(VirtualKeyShort.TAB, VirtualKeyShort.ENTER);
+
+            Logger.Default.Info("wait for document to load and UI to populate");
+            Retry.WhileNull(() => Window.FindFirstByXPath("/Pane/Pane/Pane/Pane/Pane/Pane/Pane"), timeout: TimeSpan.FromMinutes(1), ignoreException: true);
+            var iconsAndMainArea = Window.FindFirstByXPath("/Pane/Pane");
+            var iconsAndMainAreaChildren = iconsAndMainArea.FindAllChildren();
+            var mainArea = iconsAndMainAreaChildren[0];
+            var icons = iconsAndMainAreaChildren[1];
+            var statusBar = iconsAndMainAreaChildren[2];
+            var mainAreaChildren = mainArea.FindAllChildren();
+            var messagesAreaAndTabBar = mainAreaChildren[0];
+            var tablesAndFactView = mainAreaChildren[1];
+            var messagesArea = messagesAreaAndTabBar.FindFirstChild();
+            var tablesAndFactViewChildren = tablesAndFactView.FindAllChildren();
+            var factViewAndTabBar = tablesAndFactViewChildren[0];
+            var tablesAndTabBar = tablesAndFactViewChildren[1];
+            var factView = factViewAndTabBar.FindFirstChild();
+            var tables = tablesAndTabBar.FindFirstChild();
+
+            Logger.Default.Info("copy structure to clipboard");

```

Comment:
What structure? The presentation tree?

Reply:
want to update the log to say its grabbing the xbrl structure or presentation structure or something?